src/js/media-container.js

```diff
@@ -348,6 +350,8 @@ class MediaContainer extends window.HTMLElement {
     // Unhide for keyboard controlling
     this.addEventListener('keyup', (e) => {
       scheduleInactive();
+
+      this.handleKeyupEvent(e);
```

Comment:
suggestion(blocker): Before merging/releasing, let's make sure there's some way for implementors to disable this functionality. Can be a simple attribute like `disableshortcuts` (or whatever name is appropriate). Should also be an `observedAttribute` to ensure that the relevant event handler(s) are added/removed.

Reply:
There doesn't seem to be a very good precedent for this, but I would lean towards `nokeyboardshortcuts` or `noshortcuts`, potentially with `globalkeyboardhosrtcuts` or `globalshortcuts` for later attaching to `window`.

Reply:
https://github.com/muxinc/media-chrome/pull/263#discussion_r919492637